lib/travis/build/script/go.rb

```diff
@@ -196,6 +200,21 @@ def update_gimme
             sh.cmd %Q'gimme #{DEFAULTS[:go]} &>/dev/null'
           end
 
+          def fetch_dep
+            dep = "$HOME/gopath/bin/dep"
+
+            sh.mkdir "$HOME/gopath/bin", echo: false, recursive: true
+
+            sh.if "$TRAVIS_OS_NAME = osx" do
+              sh.cmd "curl -sL -o #{dep} https://github.com/golang/dep/releases/download/v0.3.2/dep-darwin-amd64", echo: false
```

Comment:
Hard coding the version number is not wise.

Reply:
Agree, I was wondering about that in regards to my previous `app_host` question, it seems that the binaries are rehosted somewhere and then referred to there.

Reply:
There is no URL for "latest stable version". We _could_ use `master` branch, but personally, I'm against it – it will make bug reporting much harder.

Reply:
Yeah, even the homebrew formula for `dep` is just done against the newest version:

https://github.com/Homebrew/homebrew-core/blob/master/Formula/dep.rb#L4

At any rate, curious if there is a place y'all park binaries to keep them from getting downloaded from remotes. I'd love to wrap this PR up if possible, if only because it would be immediately useful for my project, hah.